### PR TITLE
[agent] add certificatesigningrequests 

### PR DESCRIPTION
**Pull request by fremmi**
## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Title of the PR starts with chart name (e.g. [mychartname])
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md (or README.tpl in some charts)
- [ ]  Check GithubAction checks (like lint) to avoid merge-check stoppers

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

**Review by yashgiri — CHANGES_REQUESTED** [review]
For file charts/sysdig-deploy/Chart.yaml

agent chart version needs to be updated.

Also sysdig-deploy changelog needs to be updated

File: charts/agent/Chart.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: Sysdig Monitor and Secure agent
 type: application
 
 # currently matching sysdig 1.14.32
-version: 1.5.21
+version: 1.5.22
 
 appVersion: 12.8.1
```
Comment (yashgiri) [inline charts/agent/Chart.yaml:11]: For this file version of the agent over here : https://github.com/sysdiglabs/charts/blob/c1b2c8740db8f2c6b180cfdb0e12da1f4aa134af/charts/sysdig-deploy/Chart.yaml#L23
will need to be changed.
Edit: For some reason I cannot add the same as a suggestion.

**Review by yashgiri — CHANGES_REQUESTED** [review]

**Review by irozzo-1A — APPROVED** [review]
LGTM

**Review by yashgiri — APPROVED** [review]

**Review by irozzo-1A — APPROVED** [review]
LGTM

**Pull request merged**